.eslintrc.yml

```diff
@@ -0,0 +1,12 @@
+env:
+  es6: true
+  node: true
+  mocha: true
+extends:
+  - eslint:recommended
+  - airbnb-base
+  - plugin:import/errors
+  - plugin:import/warnings
+rules:
+  "comma-dangle": ["error", "never"]
+  
```

Comment:
EOF (strange..., check that you don't have spaces or something similar on the last line.